setup.py

```diff
@@ -2,12 +2,23 @@
 
 from setuptools import find_packages, setup
 
-requirements = [
+# NOTE(cutwater): Because bindings are statically generated, requirements list
+#   from pulp-galaxy/setup.py has to be copied here and manually maintained.
```

Comment:
Would it make any sense to add a setup.py to galaxy_pulp and install as it's own package?

(Probably not worth it considering end goal is to not have galaxy_pulp at all, so no need to change)